File: algo/70. Climbing Stairs.py
class Solution(object):
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        if n==1:
            return 1
        if n==2:
            return 2
        prev1 = 1
        prev2 = 2
        for i in range(n-2):
            temp = prev1+prev2
            prev1 = prev2
            prev2 = temp
        return prev2

File: round2/70. Climbing Stairs.cs
public class Solution {
    public int ClimbStairs(int n) {
        if (n == 1) return 1;
        if (n == 2) return 2;
        int prev1 = 1, prev2 = 2, res=0;
        for (int i=2; i<n; i++) {
            res = prev1 + prev2;
            prev1 = prev2;
            prev2 = res;
        }
        return res;
    }
}